hwilib/commands.py

```diff
@@ -32,30 +32,24 @@ def __init__(self,*args,**kwargs):
 
 # Get the client for the device
 def get_client(device_type, device_path, password=None):
-    # Open the device
```

Comment:
don't think these lines should be deleted yet in this commit?

Reply:
Moved to "Unify device client constructor to path and password "